packages/flutter_tools/lib/executable.dart

```diff
@@ -82,25 +84,35 @@ Future<Null> main(List<String> args) async {
     context[DeviceManager] = new DeviceManager();
     Doctor.initGlobal();
 
+    if (flutterUsage.isFirstRun) {
+      printStatus(
+        'The Flutter tool anonymously reports feature usage statistics and basic crash reports to improve\n'
+        'the tool over time; use "flutter config" to control this behavior. See our privacy policy:\n'
```

Comment:
"our" is unclear here. This is Google's privacy policy, not "the Flutter project's". Maybe rephrase this as:

```
The Flutter tool anonymously reports feature usage statistics and basic crash reports to Google to
help Google contribute improvements to Flutter over time. Use "flutter config" to control this behavior.
See Google's privacy policy: https://www.google.com/intl/en/policies/privacy/
```